components/display/tm1651.rst

```diff
@@ -0,0 +1,59 @@
+TM1651 Battery Display
+======================
+
+.. seo::
+    :description: Instructions for setting up TM1651 Battery Display.
+    :image: tm1651_battery_display.jpg
+
+.. _tm1651:
+
+Battery Display
+---------------
+
+The ``tm1651`` display platform allows you to use battery display units based on TM1651 chip, like
+`this one <https://aliexpress.com/item/32811491559.html>`__
+with ESPHome. Currently integration supports level and brightness setting. All updates can be made via lambda expressions.
+
+.. figure:: images/tm1651-battery-display.jpg
+    :align: center
+    :width: 70.0%
+
+.. code-block:: yaml
+
+    api:
```

Comment:
This example section should be about the minimal config to get the device working.

Remove the `api` here and move it to a separate example below the config variables section explaining how to use the tm1651